packages/material-ui-lab/src/Pagination/Pagination.js

```diff
@@ -80,19 +82,23 @@ const Pagination = React.forwardRef(function Pagination(props, ref) {
 });
 
 Pagination.propTypes = {
+  // ----------------------------- Warning --------------------------------
+  // | These PropTypes are generated from the TypeScript type definitions |
+  // |     To update them edit the d.ts file and run "yarn proptypes"     |
+  // ----------------------------------------------------------------------
   /**
    * Number of always visible pages at the beginning and end.
    */
   boundaryCount: PropTypes.number,
   /**
-   * Pagination items.
+   * The content of the component.
    */
   children: PropTypes.node,
```

Comment:
@oliviertassinari Since the demo was removed from the docs, was it your intention to remove the children prop? (Also the markdown no longer makes sense: "Pagination supports two approaches for Router integration"...)

Reply:
Thanks for the highlight on the inconsistency. From my perspective, we need a single approach. I would be in favor of removing mentions to the `children` prop.

Reply:
> From my perspective, we need a single approach.

We only had both since that was what you'd originally suggested! 😆

> I would be in favor of removing mentions to the children prop.

We can remove it altogether in that case. (And now I'm trying to remember why `<li>` ended up in the Pagination component, and not PaginationItem. I'm sure it was related, but I can't think why.)

Reply:
Sorry, it sounds like I have changed my mind. What's your perspective on it?

Reply:
> What's your perspective on it?

Let's keep it simple and just have one approach. The render-prop takes less code to use than children.